src/__tests__/looksrare/signMakerOrders.test.ts

```diff
@@ -43,23 +42,23 @@ describe("Sign maker orders", () => {
         strategyId: 1,
         collectionType: CollectionType.ERC721,
         orderNonce: 1,
-        collection: collectionERC721.address,
+        collection: await collectionERC721.getAddress(),
         currency: mocks.addresses.WETH,
         signer: signers.user1.address,
         startTime: Math.floor(Date.now() / 1000),
         endTime: Math.floor(Date.now() / 1000 + 3600),
-        price: utils.parseEther("1").toString(),
+        price: parseEther("1").toString(),
         itemIds: [1],
         amounts: [1],
         additionalParameters: encodeParams([], getMakerParamsTypes(StrategyType.standard)),
       };
 
       const signature = await lrUser1.signMakerOrder(makerOrder);
 
-      expect(utils.verifyTypedData(domain, makerTypes, makerOrder, signature)).to.equal(signers.user1.address);
+      expect(verifyTypedData(domain, makerTypes, makerOrder, signature)).to.equal(signers.user1.address);
       await expect(verifier.verifySignature(makerOrder, signature)).to.eventually.be.fulfilled;
       await expect(verifier.verifySignature(makerOrder, faultySignature)).to.eventually.be.rejectedWith(
-        "call revert exception"
+        /reverted with/
```

Comment:
With ethers v6, we had an error like `VM exception .... reverted with...`, so I updated this part